### DISCUSSION TITLE
python sdk proxy.maxVectorFieldNum how to setting ?

### DISCUSSION THREAD

**Discussion by DotaArtist** [milvus-io/milvus#38484] [Q&A and General discussion]
python sdk proxy.maxVectorFieldNum how to setting ?

**Comment by DotaArtist** [comment]
RPC error: [create_collection], <MilvusException: (code=65535, message=maximum vector field's number should be limited to 4)>

**Comment by yhmo** [comment]
proxy.maxVectorFieldNum is configured in the milvus.yaml, the default value is 4, range is [1, 10]
```
proxy:
  maxVectorFieldNum: 4 # The maximum number of vector fields that can be specified in a collection. Value range: [1, 10].
```

This doc tells how to configure milvus in standalone(by docker compose):
https://milvus.io/docs/configure-docker.md?tab=component

This doc tells how to configure milvus in cluster(by Helm)
https://milvus.io/docs/configure-helm.md?tab=component

**Reply by DotaArtist** [reply]
如果我要使用 milvus cloud 如何创建呢？

**Reply by yhmo** [reply]
zilliz cloud的话，你发个zilliz support ticket给后台问问看能否改
![Screenshot from 2024-12-17 10-08-04](https://github.com/user-attachments/assets/fdab4828-3fa5-4bce-8337-8a88ce258abf)
